benchmarks/314.hadoop/hadoop.sh

```diff
@@ -1,9 +1,6 @@
 #!/bin/bash
 
-set -x
```

Comment:
it run very long time.
if we need debug any issue, or the issue reproducibility random,
we can add it to the corresponding position.